src/plugins/github/parseReferences.test.js

```diff
@@ -0,0 +1,150 @@
+// @flow
+
+import {
+  findNumericReferences,
+  findGithubUrlReferences,
+} from "./parseReferences.js";
+import type {GithubExactUrlMatch} from "./parseReferences.js";
+
+describe("reference finding", () => {
+  it("finds no numeric references when not present", () => {
+    expect(findNumericReferences("foo bar bod boink")).toHaveLength(0);
+    expect(findNumericReferences("")).toHaveLength(0);
+  });
+
+  it("finds trivial numeric references", () => {
+    expect(findNumericReferences("#1, #2, and #3")).toEqual([1, 2, 3]);
+  });
+
+  it("finds numeric references in a multiline string", () => {
+    const example = `
+    This is a multiline string.
+    It refers to #1. Oh, and to #2 too.
+    (#42 might be included too - who knows?)`;
+    expect(findNumericReferences(example)).toEqual([1, 2, 42]);
+  });
+
+  it("does not find bad references", () => {
+    expect(findNumericReferences("foo#123 #124bar")).toHaveLength(0);
```

Comment:
The reference form `repo-owner/repo-name#number` is valid on GitHub. If you want to document that we're not doing this, then that's fine with me.

Reply:
Documented